src/advocate.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: "Seeking: Senior Developer Tools Engineer"
```

Comment:
all good, just gotta update this one line

Reply:
:grimacing: